docs/CONTRIBUTING.md

```diff
@@ -25,3 +25,38 @@ If you are new to open-source contribution, please read the this [guide](https:/
 ## Writing tests
 
 As a best practice, the project requires tests to be submitted at the same PR with the code. If you are developing a new feature, please remember writting tests for it as well. See the relevant [testing documenation](./docs/testing.md)
+
+## An example PR template
+
+If you are looking for a PR template, you can use the following. 
+After having both `/lgtm` and `/approve` labels from approvers and reviewers, your PR will be merged by `Openshift Merge Bot`
+To prevent any accidental merge, marking the PR as `work in progress` by adding the `[WIP]` tag would be nice
+
+```
+What type of PR is this? (Could be a bug/feature/cleanup/docs/design..)
+
+What this PR does / why we need it:
+
+Which issue(s) this PR fixes(can be a reference to existing issue or jira/bz):
+
+Fixes #
+
+Special notes for your reviewer:
+
+Is it a breaking change or backward compatible?
+
+Pre-checks:
+I've run
+- [ ] the `make` command and it returns 0
+- [ ] the latest changes against crc/k8s
+- [ ] the `make coverage` command to generate new calculated coverage
+```
+
+## PR Review Checks
+
+This section isn't mandatory for all PR's but it would be nice to check the followings after reviewing the source code as a post-check.
+
+- checkout the code and run the operator against your own crc/k8s, make sure the provided feature is working as expected
+- make sure that the coverage badge will be updated in the PR's context
+- run `make` and make sure there are no code changes(especially in `go.mod`, `go.sum` and for the other generated files)
```

Comment:
I think `make` will fail if there are dependencies that are not in the `vendor` folder. Also, make is executed by the CI, so no need to do that as reviewer